src/v2/Apps/Order/Routes/Review/index.tsx

```diff
@@ -122,23 +122,32 @@ export class ReviewRoute extends Component<ReviewProps> {
             }
           })
       } else {
-        const { order, router, isEigen } = this.props
-        // Buy-mode order redirects to the status page. Eigen must keep the user inside the webview.
-        if (order.mode !== "OFFER" || isEigen) {
-          return router.push(`/orders/${order.internalID}/status`)
-        }
-        // Make offer in inquiry redirects to the conversation page
-        if (order.source === "inquiry") {
-          return router.push(
-            `/user/conversations/${order.conversation?.internalID}`
-          )
-        }
-        // Make offer from artwork page redirects to the artwork page
+        const { order, router, isEigen, featureFlags } = this.props
+        const isCBNEnabled =
+          featureFlags?.["conversational-buy-now"]?.flagEnabled
```

Comment:
I found no examples of unleash's feature flags implemented on class components. I'm reading the `featureFlags` added by the `withSystemContext()`